README.md

```diff
@@ -51,10 +51,17 @@
     <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
   </ul>
 </details>
```

Comment:
- [ ] Your project doesn't make use of `postgresql or expressJs` and as such I kindly suggest you remove them from the tech stacks as it is not a true representation of the technologies used in building this project 👍 ✨